src/ClientData/ScopeIdProvider.cpp

```diff
@@ -40,7 +40,8 @@ std::unique_ptr<NameEqualityScopeIdProvider> NameEqualityScopeIdProvider::Create
 }
 
 uint64_t NameEqualityScopeIdProvider::ProvideId(const TimerInfo& timer_info) {
-  if (timer_info.function_id() != orbit_grpc_protos::kInvalidFunctionId) {
+  if (timer_info.function_id() != orbit_grpc_protos::kInvalidFunctionId &&
+      timer_info.type() == orbit_client_protos::TimerInfo_Type::TimerInfo_Type_kNone) {
```

Comment:
kApiScope is also a thread track timer, so I don't like kThreadTrackTimer.
@florian-kuebler @dpallotti can you confirm kNone is used for hooked function only?

Reply:
I would rather not rename that now. The name was confusing since forever. The other type names are also confusing. The only valid solution is having proper classes for the different types of timers. And we are pretty close to that.